Sources/PluginLibrary/plugin.pb.swift

```diff
@@ -126,7 +126,7 @@ public struct Google_Protobuf_Compiler_CodeGeneratorRequest: SwiftProtobuf.Messa
 
   public func _protoc_generated_isEqualTo(other: Google_Protobuf_Compiler_CodeGeneratorRequest) -> Bool {
     if fileToGenerate != other.fileToGenerate {return false}
-    if (parameter != other.parameter) {return false}
+    if ((_parameter != nil || other._parameter != nil) && (_parameter == nil || other._parameter == nil || _parameter! != other._parameter!)) {return false}
```

Comment:
I might be missing something... but do we need all the nil checks? Wouldn't `_parameter != other._parameter` be sufficient?

* If both are nil/unset, they're equal
* If both are set, it'll compare the values
* If one is nil and the other isn't, it doesn't matter what the value is, they're not equal because we ignore default values